### PR TITLE
Add initial build scripts and webpack config.

### PR DESCRIPTION
**Pull request by FatJoe-Julian**

### CODE REVIEW

File: .storybook/config.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+function loadStories() {
+	req.keys().forEach(filename => req(filename));
+}
+
+addDecorator(story =>
+	<div style={{ padding: "8px" }}>
+		{story()}
+	</div>
+);
```
Comment (fatjoeluke) [inline .storybook/config.js]: No need for this decorator

It was only added to combat bootstraps reset sheet

File: .storybook/preview-head.html [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" />
+<link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"
+/>
```
Comment (fatjoeluke) [inline .storybook/preview-head.html]: This file can be removed.

Was only added to get styles into storybook

File: package.json [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    "prepublish": "npm run build",
+    "storybook": "start-storybook -p 6006"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/FatJoeWho/react-calendar.git"
+  },
+  "files": [
+    "dist",
+    "src"
```
Comment (fatjoeluke) [inline package.json]: Need to also push the README.md, a contribution guide and a style guide

File: webpack.config.js [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+});
+
+module.exports = {
+	entry: entries,
+	output: {
+		filename: "[name].js",
+		path: path.join(__dirname, "dist/components/"),
+		libraryTarget: "commonjs2"
+	},
+	externals(context, request, callback) {
```
Comment (fatjoeluke) [inline webpack.config.js]: What does this block do?

File: package.json (hunk 3 above)
Comment (fatjoeluke) [inline package.json]: https://github.com/FatJoeWho/react-calendar/community

File: webpack.config.js (hunk 4 above)
Comment (FatJoe-Julian) [inline webpack.config.js]: Earlier in the config a series of files are setup from ./src/components.
When files are not in this array treat it as external.
Externals can be set via a function.
[https://webpack.js.org/configuration/externals/#function](url)

Comment (fatjoeluke) [inline webpack.config.js]: Okay, whats the benefit of treating them as external?

**Pull request merged**